server/src/main/java/com/octopus/teamcity/opentelemetry/server/helpers/NullOTELHelperImpl.java

```diff
@@ -0,0 +1,43 @@
+package com.octopus.teamcity.opentelemetry.server.helpers;
+
+import io.opentelemetry.api.trace.Span;
+
+public class NullOTELHelperImpl implements OTELHelper {
```

Comment:
🤔 Whats the purpose of this class?

Reply:
It's used when a project does not have OTEL enabled.